### ISSUE TITLE
Add Engineer Area for Markings Service Area SRs

### ISSUE THREAD

**Issue by dianamartin** [cityofaustin/atd-knack-signs-markings#282]
**Page:** 
- [x] Markings Service Request page
- [ ] Service Request Details page
- [ ] Work Order/Job Details page
- [ ] Job Details page

We have the Engineer Area coming in automatically, I think we just have to update the script that is running for this. The CSR Display Label is on the Markings Service Request, but it's just not showing the Engineer Area in the concatenated name.

**Comment by dianamartin** [comment]
@johnclary Do you need to set something up for this too?

**Comment by johnclary** [comment]
um, i believe we get the sign engineer areas from 311. are the markings engineer areas visible on SRs in the CSR system currently?

**Comment by dianamartin** [comment]
@johnclary The engineer service areas are not coming from 311 for the 311 Markings SRs. However, we did have this enabled for the Signs SRs. It's possible we need to configure on the 311 side?

**Comment by johnclary** [comment]
yes. i believe this process involves sending a shapefile to 311 folks. it would be nice if the schema was the same as the signs engineering areas

**Comment by amenity** [comment]
@dianamartin call Chandra plz.

**Comment by dianamartin** [comment]
Bill S. was assigned to help. We emailed back and forth and I told him that we couldn't see his test SR and to sent it in PROD since it won't mess up anything with Knack. 

I saw it in the system and emailed him back.
>I was able to see the SR 20-00016442 come over. It’s working, I was able to see the service are for CENTRAL. 
![image.png](https://images.zenhubusercontent.com/5cb4c11373d7e234297de5cf/d496ab0a-1975-4a4c-96a0-ceae0420535b)
![image.png](https://images.zenhubusercontent.com/5cb4c11373d7e234297de5cf/76ab3cfa-bf12-447e-b97f-91c774e0b581)
